tsconfig.json

```diff
@@ -1,16 +1,23 @@
 {
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "sourceMap": true,
+    "noImplicitAny": false,
```

Comment:
if set to false, there will be too much error to fix for now, will comeback to this later